WordPress/Classes/ViewRelated/Me/Me Main/Presenter/MeScenePresenter.swift

```diff
@@ -42,20 +43,23 @@ private extension MeScenePresenter {
     func makeNavigationController() -> UINavigationController {
         let meController = makeMeViewController()
         let navigationController = UINavigationController(rootViewController: meController)
-        meController.navigationItem.leftBarButtonItem = makeDoneButton()
+        navigationController.restorationIdentifier = Restorer.Identifier.navigationController.rawValue
+        meController.navigationItem.rightBarButtonItem = makeDoneButton()
+        // present in formSheet on iPad, default on iPhone
+        if WPDeviceIdentification.isiPad() {
+            navigationController.modalPresentationStyle = .formSheet
+            navigationController.modalTransitionStyle = .coverVertical
+        }
```

Comment:
From what I can tell, this check isn't necessary since `.formSheet` automatically adjusts depending on the trait environment.

```suggestion
        navigationController.modalPresentationStyle = .formSheet
        navigationController.modalTransitionStyle = .coverVertical
```